src/sass/components/header.sass

```diff
@@ -1,29 +1,151 @@
+// Common
 .navbar
   margin: 0 auto
   position: relative
+  height: 100%
+
   .button.navbar-burger
     border: none
     background-color: transparent
     outline: none
-    color: $text
+    color: $sub-text
+    height: 60px
+    padding: 0 0.5rem
+    width: 46px
+
+    span
+      height: 2px
+      margin-left: -0.5rem
+      border-radius: 1px
+
+  .navbar-item,
+  .navbar-link
+    transition: color $speed $easing
+
+  .navbar-link::after
+    display: none
+
+  .has-dropdown .navbar-link::after
+    border-left: 0.3rem solid transparent
+    border-right: 0.3rem solid transparent
+    border-bottom: 0.3rem solid #818D9A
+    border-top: 0
+    content: " "
+    display: block
+    height: 0.3rem
+    pointer-events: none
+    position: absolute
+    transform: rotate(180deg)
+    width: 0.3rem
+    margin-top: -0.3em
+    right: 1.125em
+    top: 50%
+    transition: transform $speed $easing
+
+.navbar-primary
+  min-height: 60px
+
+// mobile
++until($desktop)
+  .section,
+  .container
+    padding: 0
+
+  .navbar
+    .navbar-menu
+      padding: 0
+      border-bottom: 2px solid #D2DCE8
```

Comment:
We could use color variables in case like this. Where are more places to change. Down the code.

Reply:
Good point